cmd/job-executor-service/main.go

```diff
@@ -48,13 +46,15 @@ type envConfig struct {
 	// Default resource requests memory for job and init container
 	DefaultResourceRequestsMemory string `envconfig:"DEFAULT_RESOURCE_REQUESTS_MEMORY"`
 	// Respond with .finished event if no configuration found
-	AlwaysSendFinishedEvent string `envconfig:"ALWAYS_SEND_FINISHED_EVENT"`
+	AlwaysSendFinishedEvent bool `envconfig:"ALWAYS_SEND_FINISHED_EVENT"`
```

Comment:
I'm wondering if there was a reason why we used `string` here :D 
do we know that `bool` works reliably with envconfig?